src/App.css

```diff
@@ -49,15 +46,53 @@ tfoot {
   padding: 10px;
 }
 
-tr:nth-child(even) {
+#userTable tr:nth-child(even) {
   background-color: rgb(215, 215, 215);
 }
 
 .top-panel__element {
   padding: 15px;
   width: 300px;
   font-size: 18px;
+  background-color: transparent;

```

Comment:
Not now but we need to fix the css of the project or this will get out of hand quickly.

Reply:
Absolutely!

Reply:
Yea i was curious if we should focus too much on that for this POC. Once we have the real application we're going to be using John's styles so this likely wont be an issue